options.go

```diff
@@ -14,7 +14,7 @@ type Option func(*config) error
 
 func UseTreeBitWidth(bitWidth uint) Option {
 	return func(c *config) error {
-		if bitWidth < 1 {
+		if bitWidth <= 1 {
 			return fmt.Errorf("bit width must be at least 2, is %d", bitWidth)
```

Comment:
Can you fix the error as well? (and maybe clarify with something like "must be at least 1 (e.g., 2 children)").